include/mc_tasks/SplineTrajectoryTask.h

```diff
@@ -141,6 +141,25 @@ struct SplineTrajectoryTask : public TrajectoryTaskGeneric<tasks::qp::TransformT
    */
   unsigned displaySamples() const;
 
+  /**
+   * @brief Allows to pause the task
+   *
+   * @warning Pausing sets the task's desired velocity and acceleration to zero, which
+   * will suddently stop the motion. Avoid pausing during high-speed
+   * trajectories.
+   *
+   * @param paused True to pause the task, False to resume.
+   */
```

Comment:
Resuming the trajectory also resumes it at full speed. That's probably worth mentioning :)